setup.py

```diff
@@ -31,6 +31,7 @@
         ],
         'dolphin.storage.drivers': [
             'fake_storage fake_driver = dolphin.drivers.fake_storage:FakeStorageDriver',
+            'huawei oceanstor = dolphin.drivers.huawei.oceanstor.oceanstor:OceanStorDriver',
             'dellemc vmax = dolphin.drivers.dell_emc.vmax.vmax:VMAXStorageDriver'
         ]
```

Comment:
except fake_storage, the other storage should be placed in alphabetical order

Reply:
Done